app/views/necesidad/necesidades.html

```diff
@@ -2,58 +2,50 @@
   <h2 class="page-header text-center">{{ 'GESTION_NECESIDADES' | translate }}</h2>
   <div class="panel-body">
     <div class="row">
-        <div class="panel panel-default">
-          <div class="panel-heading">
-            <h4 class="intro-text"> {{ 'LISTA_NECESIDADES' | translate }} </h4>
-          </div>
-          <div class="panel-body">
-              <div ui-grid="necesidades.gridOptions" ui-grid-selection ui-grid-pagination class="grid"></div>
-          </div>          
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <h4 class="intro-text"> {{ 'LISTA_NECESIDADES' | translate }} </h4>
+        </div>
+        <div class="panel-body">
+          <div ui-grid="necesidades.gridOptions" ui-grid-selection ui-grid-pagination class="grid"></div>
         </div>
       </div>
-
+    </div>
   </div>
 </div>
 
 <div id="myModal" class="modal fade" role="dialog">
-
   <!-- Modal content-->
   <div class="modal-content col-md-10 col-md-offset-1" style="margin-top: 5%">
     <div class="modal-body" ng-if="necesidades.g_necesidad!=undefined">
-      <visualizar-necesidad necesidad="necesidades.necesidad" vigencia="necesidades.vigencia" numero="necesidades.numero_el" estado="necesidades.g_necesidad.EstadoNecesidadId" modalidadSel="necesidades.modalidadSel" tipoContrato="necesidades.tipoContrato">
+      <visualizar-necesidad necesidad="necesidades.necesidad" numero="necesidades.numero_el" estado="necesidades.g_necesidad.EstadoNecesidadId" modalidadSel="necesidades.modalidadSel" tipoContrato="necesidades.tipoContrato">
       </visualizar-necesidad>
```

Comment:
Por legibilidad, de una vez separar cada propiedad en una línea aparte:
```suggestion
      <visualizar-necesidad
      necesidad="necesidades.necesidad"
      numero="necesidades.numero_el"
      estado="necesidades.g_necesidad.EstadoNecesidadId"
      modalidadSel="necesidades.modalidadSel"
      tipoContrato="necesidades.tipoContrato"
      ></visualizar-necesidad>
```